CONTRIBUTING.md

```diff
@@ -143,7 +143,7 @@ with making the reviews as straightforward as possible:
   large to review in one sitting, or if changes are requested, then the
   maintainer needs to repeatedly re-read code that has already been considered.
 * The commit history is important. This is a large codebase, developed over many
-  years by many developers. We frequently need to read the commit history (e.g.
+  years by many developers. We frequently need to read the commit history (e.g.,
```

Comment:
```suggestion
  years by many developers. We frequently need to read the commit history (for example
```

Reply:
Yes that's fine, can you fold that into the existing commit rather than having me commit it as a fixup? Probably change the title of the commit as well to remove the punctuation reference.